### PR TITLE
Two fixes for spec.sh

### PR DESCRIPTION
**Pull request by eriknordmark**
First commit is a wrong ztype.

Second commit is to detect iommu_group overlap with devices we do not care about (to avoid disabling some random memory timing controller or whatever might sit on the PCI bus) TBD: Once PR #2181 makes it to master I'll update its description to describe this.

Second commit also has a fix for an extra "}," after the cost which produces invalid json.

Third commit is just a comment about a TBD for the networking adapters sitting on USB, which the script currently doesn't put in the same assignment group as the USB controller.

### CODE REVIEW

File: pkg/debug/spec.sh [hunk 1]
```diff
@@ -37,9 +59,83 @@ pci_iommu_group() {
...
+        local ifname
+        local ztype
+        ifname=$(ls "/sys/bus/pci/devices/${pci}/net")
+        ztype=1
+        if [ "${ifname:0:4}" = "wlan" ]; then
+            ztype=5
+        elif [ "${ifname:0:4}" = "wwan" ]; then
+            ztype=6
+        fi
+    elif lspci -D -s "${pci}" | grep -q USB; then
```
Comment (rvs) [inline pkg/debug/spec.sh:109]: This hardcoding of USB, Audio, VGA seems a bit suspect.

**Review by rvs — APPROVED** [review]
Mostly looks good -- I just don't know if it is time to split this into multiple scripts

File: pkg/debug/spec.sh (hunk 1 above)
Comment (eriknordmark) [reply pkg/debug/spec.sh:109]: The script already has those strings hardcoded elsewhere.

**Review by rvs — APPROVED** [review]
LGTM

**Comment by eriknordmark** [comment]
Need to update the markdown file as well.

**Pull request merged**